### PR TITLE
Add email contact

### PR DESCRIPTION
**Pull request by metageeky**

### CODE REVIEW

File: forms/alt-collection.html [hunk 1]
```diff
@@ -61,6 +61,13 @@ <h2>How to Write a Speaker Image Description</h2>
 			</li>
 		</ul>
 		
+		<p>
+			If you have any questions on writing your speaker image alt text, please send your questions to <a style="text-decoration: underline" href="mailto:code4lib.accessibility@gmail.com">code4lib.accessibility AT gmail.com</a>.
+		</p>
+
+
+If you did not grant access, you should check this activity and
```
Comment (dzoladz) [inline forms/alt-collection.html:69]: What's happening in line 69?

File: forms/alt-collection.html [hunk 2]
```diff
@@ -61,6 +61,13 @@ <h2>How to Write a Speaker Image Description</h2>
 			</li>
 		</ul>
 		
+		<p>
+			If you have any questions on writing your speaker image alt text, please send your questions to <a style="text-decoration: underline" href="mailto:code4lib.accessibility@gmail.com">code4lib.accessibility AT gmail.com</a>.
```
Comment (phette23) [inline forms/alt-collection.html:65]: We have a datum for the accessibility email, so the email references could be `{{ site.data.conf.accessibility-email }}`. Though it looks like the gmail address is commented out and a googlegroups one is used? Should we be using the other one? See https://github.com/code4lib/2022.code4lib.org/blob/main/_data/conf.yml#L109-L110

File: forms/alt-collection.html (hunk 1 above)
Comment (phette23) [inline forms/alt-collection.html:69]: Looks like this sentence was truncated?

**Review by phette23 — CHANGES_REQUESTED** [review]
See a couple comments. We should try to use the accessibility email data point and then the sentence just before the form got cut off.

**Comment by metageeky** [comment]
The email and the group are different things. For this purpose, using the email is more appropriate. This is not debatable.

**Comment by metageeky** [comment]
I push up a fix for that bad text (copy and paste I did not expect).

**Review by phette23 — APPROVED** [review]

**Pull request merged**